src/component/Form.js

```diff
@@ -0,0 +1,31 @@
+import './styles/form.css';
+import { useDispatch, useSelector } from 'react-redux';
+import { addBook } from '../redux/book/booksSlice';
+
+export default function Form() {
+  const dispatch = useDispatch();
+  const { totalbooks } = useSelector((state) => state.book);
+  const book = () => {
+    const tit = document.querySelector('.title').value;
+    const auth = document.querySelector('.author').value;
+    const i = totalbooks.length + 1;
+    const b = {
+      title: tit,
+      author: auth,
+      item_id: `item${i}`,
+    };
+    document.querySelector('.title').value = '';
+    document.querySelector('.author').value = '';
+    return b;
+  };
+  return (
+    <form>
+      <span>Add new book</span>
+      <div>
+        <input className="title" placeholder="book title" />
+        <input className="author" placeholder="author" />
+        <button type="button" onClick={() => dispatch(addBook(book()))}>ADD BOOK</button>
+      </div>
+    </form>
+  );
+}
```

Comment:
- [ ] Your form component should be controlled. This means the value of the title and author should be set using a local state, not the user's direct input; and the value of the states should be using the `onChange` method. Kindly see [this article](https://blog.logrocket.com/controlled-vs-uncontrolled-components-in-react/) on how to make controlled components.